tools/topology/m4/pcm.m4

```diff
@@ -88,9 +88,20 @@ define(`PCM_PLAYBACK_ADD',
 `}', `fatal_error(`Invalid parameters ($#) to PCM_PLAYBACK_ADD')')'
 )
 
-dnl PCM_CAPTURE_ADD(name, pcm_id, capture)
-define(`PCM_CAPTURE_ADD',
-`ifelse(`$#', `3',
+dnl PCM_CAPTURE_ADD_COMMON(name, pcm_id, capture, lp)
```

Comment:
Wont this change impact all capture PCMs ? (i.e. all users of PCM_CAPTURE_ADD)

Reply:
this is only PCM_CAPTURE_ADD_COMMON part, all users of PCM_CAPTURE_ADD are still using the same interface.